apps/www/components/OpenSourceSection.tsx

```diff
@@ -0,0 +1,70 @@
+import Link from 'next/link'
+import { Button } from 'ui'
+import SectionContainer from './Layouts/SectionContainer'
+import { useSendTelemetryEvent } from '~/lib/telemetry'
+import staticContent from '.generated/staticContent/_index.json'
+
+const OpenSourceSection = () => {
+  const sendTelemetryEvent = useSendTelemetryEvent()
+  const githubStars = staticContent.githubStars
+
+  const kFormatter = (num: number) => {
+    const kFormat = Math.floor(num / 1000)
+    const lastTwoDigits = num % 1000
+
+    const decimalPart = Math.floor((lastTwoDigits % 100) / 10)
+    const hundreds = Math.floor(lastTwoDigits / 100)
+
+    const isAlmostNextThousand = decimalPart >= 8 && hundreds >= 9
+
+    const showDecimals =
+      (!isAlmostNextThousand && hundreds >= 1) || (hundreds === 0 && decimalPart >= 8)
+
+    return showDecimals
+      ? `${kFormat}.${decimalPart >= 8 ? hundreds + 1 : hundreds}K`
+      : `${isAlmostNextThousand ? kFormat + 1 : kFormat}K`
+  }
+
+  return (
+    <SectionContainer className="w-full text-center flex flex-col items-center">
+      <h2 className="h2">Open source from day one</h2>
+      <p className="p max-w-xl">
+        Supabase is built in the open because we believe great developer tools should be
+        transparent, inspectable, and owned by the community. Read, contribute, self-host. You're
+        never locked in, and always in control.
+      </p>
+      <div className="mt-8 flex flex-col items-center gap-2">
+        <Button
```

Comment:
Minor mit: the left-side of the button looks a bit imbalanced with the right-side because of the GitHub logo. Could do something like `className="!pl-2.5"` on the `Button` to optically align it.

Not a big deal, feel free to ignore.